src/ansys/dpf/composites/server_helpers/_load_plugin.py

```diff
@@ -48,17 +51,19 @@ def get_lib_from_name(name: str) -> str:
             "nt": ["dpf", "plugins", "dpf_composites"],
             "posix": ["dpf", "plugins", "dpf_composites"],
         },
-        "Ans.Dpf.EngineeringData": {
-            "nt": ["dpf", "bin", "winx64"],
-            "posix": ["dpf", "bin", "linx64"],
-        },
     }
 
+    if not ansys_path:
+        ansys_path = server.ansys_path if server.ansys_path else AWP_ROOT_DOCKER
```

Comment:
I think this will also use AWP_ROOT_DOCKER root if we connect to a server that does not run in a docker container. Probably we should add a is_docker argument to connect_to_or_start_server